hepdata/modules/records/views.py

```diff
@@ -744,6 +767,7 @@ def get_resource(resource_id):
                     status_code = 404 if 'error' in ctx['json_ld'] else 200
                     return jsonify(ctx['json_ld']), status_code
                 else:
+                    if filesize: ctx['filesize'] = '%.2f'%((filesize / 1024) / 1024) # Set filesize if exists
```

Comment:
Here you can also add the threshold to the `ctx` so it can be used in the HTML message:
```python
ctx['ADDITIONAL_SIZE_LOAD_CHECK_THRESHOLD'] = '%.2f'%((ADDITIONAL_SIZE_LOAD_CHECK_THRESHOLD / 1024) / 1024)
```